lib/assessment.rb

```diff
@@ -13,28 +13,36 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(&:+)
```

Comment:
Correct. In fact, as of Ruby 2.0, you can even drop the `&` and just write
`enum.reduce(:+)`